src/Ocelot/Buffering/Constants.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Ocelot.Buffering;
+
+public class Constants
+{
+    public const string BufferedBodyItem = "BufferedBody";
```

Comment:
Indeed, it would be more effective to define it directly within the class as a public constant.

Reply:
Wasn't a big fan to reference `BufferingMiddleware.BufferedBodyItem` inside another class (as it's not really related solely to the middleware but to the whole functionnality), but not entirely conviced by adding a class with a single item. OK to move this.